lib/index.js

```diff
@@ -24,10 +24,22 @@ var serve = function(options) {
 
         fileServer.serve(request, response, function(err, res) {
           if (err) {
-            log(chalk.red("[" + err.status + "] " + request.url), true);
-
-            response.writeHead(err.status, err.headers);
-            response.end("Not found");
+            if (options.redirects[request.url]) {
+              log(chalk.yellow("[" + 301 + "] " + request.url + " > " + options.redirects[request.url]), true);
```

Comment:
That could work. I was thinking along the lines of:
```
HTP_ERRORS = {
  "not_found": 404,
  ...
}
```
so in code it's HTTP_ERRORS.not_found rather than magic numbers. There are only few, so it's not that big of a deal, I guess. I'll leave it up to you, maybe it's not required at all depending on how the error handling/matching is implemented.